packages/contracts-bedrock/src/periphery/faucet/Faucet.sol

```diff
@@ -124,4 +124,11 @@ contract Faucet {
 
         emit Drip(config.name, _auth.id, config.amount, _params.recipient);
     }
+
+    /// @notice Returns the enable status of a given auth module.
+    /// @param _module module to check.
+    /// @return bool enabled status of auth modulew.
+    function getModuleStatus(IFaucetAuthModule _module) public view returns (bool) {
```

Comment:
nit: this is nit picky but since this is actually returning whether is it enabled or not it might be better to name this `isModuleEnabled`, that way from the name you can tell exactly what it is doing. getStatus made me think it was returning a status enum.